spec/factories/recipient_families.rb

```diff
@@ -0,0 +1,26 @@
+# == Schema Information
+#
+# Table name: recipient_families
+#
+#  id                       :integer          not null, primary key
+#  organization_campaign_id :integer
+#  social_worker_id         :integer
+#  casenumber               :integer
+#  contact_last_name        :string
+#  contact_first_name       :string
+#  address                  :string
+#  city                     :string
+#  state                    :string
+#  zip                      :string
+#  phone                    :string
+#  created_at               :datetime         not null
+#  updated_at               :datetime         not null
+#
+
+FactoryGirl.define do
+  factory :recipient_family do
+    organization_campaign_id
```

Comment:
Should be `organization_campaign` to automatically create the association between the family & organization campaign